openmetadata-spec/src/main/resources/json/schema/security/credentials/awsCredentials.json

```
@@ -32,6 +32,27 @@
       "description": "EndPoint URL for the AWS",
       "type": "string",
       "format": "uri"
+    },
+    "profileName": {
+      "title": "Profile Name",
+      "description": "The name of a profile to create the boto session.",
```

Let's update the description here!

```suggestion
      "description": "The name of a profile to use with the boto session.",
```

---

Our code will not be responsible for creating Profiles rather using them. The common use-case I can think of from AWS is with custom Airflow Installations where the associated workers will have multiple profiles configured with AWS and they would like to pick one!